### PR TITLE
Winging It Live - OddContrast

### PR DESCRIPTION
**Pull request by SondraE**
## Related Issue(s)
[647](https://github.com/oddbird/oddleventy/issues/647)

## Steps to test/reproduce
- /2024/06/27/winging-it-live-03/
- /blog/ _
- /oddcontrast/

### CODE REVIEW

**Comment by SondraE** [comment]
@mirisuzanne 
I'm not sure why the winging episode I added isn't being displayed in the list of post or the list of winging it episodes.
Also I'd like to add the sponsor section at the bottom of this winging it episode post. How is that done?

**Comment by mirisuzanne** [comment]
@SondraE 

- The file was missing a `.md` extension.
- I added `sponsors: true` in the yaml data
- I removed the `Show Notes` tag, since it's already added to all Winging It posts automatically

looks good to me!

**Review by mirisuzanne — APPROVED** [review]

**Comment by jgerigmeyer** [comment]
> ## Steps to test/reproduce
> * /2024/06/27/winging-it-live-03/
> * /blog/ _(why isn't it showing up in the list?)_
> * /oddcontrast/

@SondraE I don't see anything changing on the `/oddcontrast/` page -- should there be something?

File: content/blog/wingingit/winging-it-live-03.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+---
+title: Using New Color Formats
+episode: 3
+sponsors: true
+tags:
+  - Accessibility
+  - Color
+  - Design
+  - Figma
+  - User Experience
```
Comment (jgerigmeyer) [inline content/blog/wingingit/winging-it-live-03.md:10]:
```suggestion
  - User Experience
  - OddTools
```

File: content/blog/wingingit/winging-it-live-03.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
+---
+title: Using New Color Formats
+episode: 3
```
[the remark this answers is not in the document]
Comment (SondraE) [reply content/blog/wingingit/winging-it-live-03.md:3]: This is the third episode of Winging It _Live_ (as opposed to Winging It not live). @stacyk and I talked a bit about whether Winging It Live should replace Winging It and pick up the numbering where we left off, which would make this episode number 9. On the other hand, maybe the live Winging It is a different thing and therefore this is number 3. The Live episodes have a different design style and live in a different section of our YouTube channel, but if we're not planning anymore Winging It episodes that aren't live, then there's a good argument for this being episode 9. (We could also retroactively make the live episodes use the same design style as the non-live episodes.)

I'm open to what you all think.

[the remark this answers is not in the document]
Comment (stacyk) [reply content/blog/wingingit/winging-it-live-03.md:3]: I think it should be winging it 9, as the method of recording shouldn't matter IMO. If we go on tour and do a live Live show, then we wouldn't start a new numbering.

[the remark this answers is not in the document]
Comment (jgerigmeyer) [reply content/blog/wingingit/winging-it-live-03.md:3]: I'm not looking at the YouTube channel, but I don't see any other episodes of Winging It Live on our site, nor a single mention of "Winging It Live" as a thing. So it's confusing for this to just say "on Winging It episode 3", right after "in Winging It episode 6".

**Comment by SondraE** [comment]
@stacyk @jgerigmeyer 
Oops. I thought I had pushed my latest changes, but apparently I hadn't.
You will now see the correct header image, the video added to OddContrast tool page, and the episode number updated.

File: content/blog/wingingit/winging-it-09.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
+---
+title: Using New Color Formats
+episode: 9
```
Comment (jgerigmeyer) [inline content/blog/wingingit/winging-it-09.md:3]: I'm okay with this, but I still find it a bit confusing that our website now lists episodes 1, 2, 3, 4, 5, 6, and 9. 🤷 

https://deploy-preview-648--oddleventy.netlify.app/wingingit/

**Review by jgerigmeyer — APPROVED** [review]

File: content/blog/wingingit/winging-it-09.md (hunk 3 above)
Comment (SondraE) [reply content/blog/wingingit/winging-it-09.md:3]: We just need to add Winging It Live 7 and 8. :)

File: src/images/winging-it/winging-it-9.jpg
Comment (stacyk) [inline src/images/winging-it/winging-it-9.jpg:1]: I like the LIVE gradient addition :)

**Review by jgerigmeyer — APPROVED** [review]

**Pull request merged**